specs/binding-mqtt-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/kafka/config/proxy.when.mqtt.topic.with.kafka.topic.yaml

```diff
@@ -28,11 +28,11 @@ bindings:
       - exit: kafka0
         when:
           - topic: sensor/one
-          - topic: sensor/two
         with:
-          topic: mqtt-sensors
+          messages: mqtt-sensors
       - exit: kafka0
         when:
           - topic: device/*
+          - topic: sensor/two
         with:
-          topic: mqtt-devices
+          messages: mqtt-devices
```

Comment:
Looks good, let's rename the scenario to something like `proxy.when.topic.with.messages.yaml` as we no longer have the collision between `mqtt` topic and `kafka` topic in the config.